docusaurus/docs/guides/deployments/10-controller/10-deploy.mdx

```diff
@@ -1,45 +1,42 @@
 ---
-id: controller
 title: Controller Deployment
 sidebar_label: Controller
+id: deploy
 ---
 
-import CliLogFormats from '../../_cli-log-levels-and-formats.md'
+import CliLogFormats from '../../../_cli-log-levels-and-formats.md'
 
-This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide. [Kubernetes is covered separately](/guides/kubernetes/hosting/kubernetes-controller.mdx).
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide.
 
 ## Installation
 
-The controller package is installed the same way as the `ziti` CLI and depends on the `openziti` package the provides the CLI.
+The controller package is installed the same way as the `ziti` CLI and depends on the `openziti` package that provides the CLI.
 
-The signed package sources are found [in the Linux downloads tab](/downloads.mdx?os=Linux).
-
-## Bootstrapping
-
-Bootstrapping happens automatically unless you disable it before running the controller service. Bootstrapping a controller involves:
-
-1. Generating a PKI
-1. Generating a configuration YAML file
-1. Initializing a database with a default admin password
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-controller
+```
 
-At a minimum, you must set these to start the service with bootstrapping enabled (the default).
+You must provide these two answers to start the service.
 
 1. In `/opt/openziti/etc/controller/env` (the systemd env file for the controller service)
     1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller
-    1. Temporarily set `ZITI_PWD` to the desired management password for user `admin`
+    1. Temporarily set `ZITI_PWD` to the desired management password for user `admin`. You may delete this password after bootstrapping for security.

```

Comment:
this shouldn't be optional. it should be mandatory. i'm still not sold on the whole bootstrapping of the environment for the user. i still dislike the idea, on the whole, for this usecase.

Reply:
I still keep coming back to a three step process:
* step 1 install the package
* step 2 bootstrap the package/install
* step 3 set it up to run it as a service

Reply:
> this shouldn't be optional. it should be mandatory. i'm still not sold on the whole bootstrapping of the environment for the user. i still dislike the idea, on the whole, for this usecase.

It is mandatory.